xclim/indices/run_length.py

```diff
@@ -11,6 +11,7 @@
 from typing import Optional, Sequence, Tuple, Union
 from warnings import warn
 
+import numba as nb
```

Comment:
I know this is a nitpick, but...
```suggestion
from numba import njit
```